Sources/Network/Parse.swift

```diff
@@ -55,4 +56,14 @@ public enum Parse {
 
         return image
     }
+
+    /// Parses empty data
+    ///
+    /// - Parameter data: Raw data
+    /// - Throws: A Parse.Error that can be of type
```

Comment:
incomplete comment 😆 (if we keep the error)

Reply:
Fixed